features/backend/product_archetypes.feature

```diff
@@ -76,6 +76,7 @@ Feature: Product archetypes
         Then I should be on the product archetype index page
         And I should see "Archetype has been successfully created."
 
+    @javascript
     Scenario: Creating a product by building archetype
```

Comment:
Is this title still appropriate? `Creating a product with attribute by building archetype` @Zales0123 WDYT?